staging/src/k8s.io/kubectl/pkg/cmd/set/set_selector.go

```diff
@@ -163,7 +165,27 @@ func (o *SetSelectorOptions) RunSelector() error {
 
 	return r.Visit(func(info *resource.Info, err error) error {
 		patch := &Patch{Info: info}
+
+		if len(o.resourceVersion) != 0 {
+			// ensure resourceVersion is always sent in the patch by clearing it from the starting JSON
+			accessor, err := meta.Accessor(info.Object)
+			if err != nil {
+				return err
+			}
+			accessor.SetResourceVersion("")
+		}
+
 		CalculatePatch(patch, scheme.DefaultJSONEncoder(), func(obj runtime.Object) ([]byte, error) {
+
+			if len(o.resourceVersion) != 0 {
+				// ensure resourceVersion is always sent in the patch by clearing it from the starting JSON
```

Comment:
Is this comment copy/paste error or should line 186 be:
```
accessor.SetResourceVersion("")
```
?

Reply:
copy/paste, fixed